### ISSUE TITLE
Request to add Anima/Quality filter limits by mission type.

### ISSUE THREAD

**Issue by gheldean** [teelolws/TLDRMissions#107]
**TLDR Version of your idea**
TLDR version: Quality/Anima cost filters by mission type

**Slightly longer version**
Can you add Anima/Quality filters for the Gold/Crafting Material/Reputation Missions?  
Examples:
 - The gold missions for 50 anima reward the same amount of gold as the 10 anima missions, and are MUCH less efficient for self sustaining tables, I typically only run the 10 anima gold missions.
  - The Crafting Material missions with green bags are also less efficient to run than the blue quality bags due to the lesser mats (this one I'm less sure of, but I've been avoiding the green bags for a long time now)
  - For Reputation missions, the 10 anima ones give 375 rep and the 50 (150?) anima ones give 2x 250 rep, WAY less efficient.

**Additional context**

**Comment by teelolws** [comment]
Hi, there IS an anima filter. Its been in the last few versions, so if you're not seeing it, you haven't updated in a while!

![image](https://user-images.githubusercontent.com/49792789/165750017-18ca289c-5aab-4bd2-8ff2-bb1ed79e3570.png)

**Comment by gheldean** [comment]
> Hi, there IS an anima filter. Its been in the last few versions, so if you're not seeing it, you haven't updated in a while!
> 
> ![image](https://user-images.githubusercontent.com/49792789/165750017-18ca289c-5aab-4bd2-8ff2-bb1ed79e3570.png)

Correct, but that's global.  I'm looking to filter Gold/Reputation Missions that are not more than 10, but for crafting/rune missions, those can go to 50.

**Comment by teelolws** [comment]
Hmm. What you're asking for might complicate the UI a bit - and its already quite overstuffed with options. I'll think about it. Might be able to add another dropdown or something.

**Comment by gheldean** [comment]
Thanks! Not a massive deal, but based of my use case :)

Thank you for this addon!

**Comment by Nillx** [comment]
Is it possible to let us select which type of anima missions we start? I would like to do the blue ones (5x35anima) and skip the green ones (15x5
![InkedScreenshot 2022-05-01 143529_LI](https://user-images.githubusercontent.com/34476875/166144117-8acfced3-8ca5-4a8f-af24-900fb267dc7b.jpg)
)

**Comment by teelolws** [comment]
Yes this is my current thought process. Another dropdown next to Anima + Crafting where you can filter by rarity of the reward.

Its down on my list. I'm still getting failed missions every day that take up most of the time I can devote to this addon, and accuracy is the highest priority.

**Comment by Nillx** [comment]
That was my thought as well. No rush and failed missions are more important than this

**Comment by gheldean** [comment]
> Is it possible to let us select which type of anima missions we start? I would like to do the blue ones (5x35anima) and skip the green ones (15x5)
> 
> ![InkedScreenshot 2022-05-01 143529_LI](https://user-images.githubusercontent.com/34476875/166144117-8acfced3-8ca5-4a8f-af24-900fb267dc7b.jpg)

Out of curiosity, all of the anima missions are net positive when successful, why skip the green ones?

**Comment by Nillx** [comment]
Because i dont have all followers unlocked on my alts and id prefer to do the blue ones only.

**Comment by gheldean** [comment]
> Because i dont have all followers unlocked on my alts and id prefer to do the blue ones only.

Oh, that makes sense, my bad.  One of the first things I do on an alt after hitting 60 is farming torghast for about an hour to fill the ranks.  A couple only have 18-20, but most have the full 22

**Comment by Nillx** [comment]
> > Because i dont have all followers unlocked on my alts and id prefer to do the blue ones only.
> 
> Oh, that makes sense, my bad. One of the first things I do on an alt after hitting 60 is farming torghast for about an hour to fill the ranks. A couple only have 18-20, but most have the full 22

Does it matter what torgahst level?

**Comment by teelolws** [comment]
> Oh, that makes sense, my bad. One of the first things I do on an alt after hitting 60 is farming torghast for about an hour to fill the ranks. A couple only have 18-20, but most have the full 22

The "meta" strategy is to get a single follower to 60 before you recruit the rest of them, so that you have level 60 troops to carry all the rest of the followers.

**Comment by gheldean** [comment]
> > Oh, that makes sense, my bad. One of the first things I do on an alt after hitting 60 is farming torghast for about an hour to fill the ranks. A couple only have 18-20, but most have the full 22
> 
> The "meta" strategy is to get a single follower to 60 before you recruit the rest of them, so that you have level 60 troops to carry all the rest of the followers.
> 
> > Does it matter what torgahst level?
> 
> On a new alt, use the Korthia campaign quest "A paladins soul" to farm an easymode version of torghast that always has a follower on the first floor.



> > Oh, that makes sense, my bad. One of the first things I do on an alt after hitting 60 is farming torghast for about an hour to fill the ranks. A couple only have 18-20, but most have the full 22
> 
> The "meta" strategy is to get a single follower to 60 before you recruit the rest of them, so that you have level 60 troops to carry all the rest of the followers.
> 
> > Does it matter what torgahst level?
> 
> On a new alt, use the Korthia campaign quest "A paladins soul" to farm an easymode version of torghast that always has a follower on the first floor.

Huh, that's neat.  Haven't done a new 60 in a while, but that seems like it will help a ton !

**Comment by teelolws** [comment]
Next update:
![image](https://user-images.githubusercontent.com/49792789/168449736-3bbe4a77-bedd-446a-afe3-d9044396c578.png)

![image](https://user-images.githubusercontent.com/49792789/168449741-563fab92-0773-40d5-ab84-9524ca7ce3a2.png)

**Comment by gheldean** [comment]
Thank you, sir! Now I can go full automatic on alts :)

**Comment by Nillx** [comment]
that looks great thanks
